package.json

```diff
@@ -6,10 +6,10 @@
   "bugs": "https://issues.gpii.net/browse/GPII/component/10800/",
   "homepage": "https://wiki.gpii.net/w/Quality_Infrastructure",
   "dependencies": {
-      "d3": "3.5.17",
-      "chartAuthoring":"https://github.com/fluid-project/chartAuthoring#6c8232a28a0066694babd1cefc66949ce0845db6",
-      "flocking": "colinbdclark/Flocking#6749bee2b7041b76f494bd188b755d568332eda0",
-      "infusion": "2.0.0-dev.20160911T222604Z.b86ecbb"
```

Comment:
It was failing to install, complaining about a missing "gh-6" branch. I just pointed everything to their latest version / master branch and it seems to have worked but it needs further validation.